src/util.rs

```diff
@@ -7,3 +7,64 @@ macro_rules! regex {
         &REGEX
     }};
 }
+
+use std::{
+    iter::FromIterator,
+    collections::HashMap,
+};
+
+
+/// Helper for `translate`, does the same thing as string.maketrans in python
+/// requires that f and t are the same length, or it takes the shorter one
+pub fn maketrans(f: String, t: String) -> HashMap<u8, char> {
```

Comment:
Why not consume `&str` instead of `String`?

Reply:
Good point. `&str` makes more sense, I'll make that change